src/ORM/Behavior/TranslateBehavior.php

```diff
@@ -482,8 +482,8 @@ protected function _bundleTranslatedFields($entity)
      */
     protected function _findExistingTranslations($ruleSet)
     {
-        $target = TableRegistry::get($this->_config['translationTable']);
-        $association = $this->_table->association($target->alias());
+        $targetAlias = Inflector::slug($this->_config['translationTable']);
```

Comment:
Hm, I don't think underscore works, I tried changing it to that. My translationTables look like `Company/Plugin.I18n`, and so the associations break at some point since that underscores to `company/plugin.i18n`. I funneled all those `Inflector::slug` calls to `$this->foo` (`$targetAlias = $this->foo($table);`)

This works:
```
private function foo($table) {
    return Inflector::slug($table, '-');
}
```

This errors out with `Method Cake\ORM\Query::__toString() must not throw an exception`
```
private function foo($table) {
    return Inflector::underscore($table);
}
```

It's hard to tell where the error is happening (I think a recent core change is accidentally obfuscating errors, not sure yet).